packages/manager/apps/telecom/package.json

```diff
@@ -136,4 +136,4 @@
     "webpack": "^4.41.2",
     "webpack-merge": "^4.2.2"
   }
-}
\ No newline at end of file
+}
```

Comment:
I will remove this diff from the Pull Request (no related).

Reply:
OK